modules/bitgo/test/v2/unit/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -343,83 +301,3 @@ describe(`Cross-Chain Recovery getWallet`, async function () {
     }
   });
 });
-
-describe(`Cross-Chain Recovery BitgoPublicApi.getUnspentInfo`, async function () {
```

Comment:
These tests are no longer needed as the BitgoPublicApi class has been removed as part of this PR (not used anywhere)